README.md

```diff
@@ -35,7 +35,9 @@ In order to use the device in user space, you need to import the provided [udev
 
 ### How to use
 
-Run `yarp-omega3-server` and send RPC commands to `/yarp-omega3-server/rpc:i`.
+For the first use it might be neccesary to reboot system to connect to `yarp-omega3-server` when connected to a usb hub.
```

Comment:
can't tell you, cause I didn't "reinstall" from scratch. Has to be double checkt. Last time it was no problem to connect to a USB hub (monitor in my case).

Reply:
I this is only related to this specific configuration I would remove this line or put it somewhere else like in a section "Known problems".

Reply:
I agree with you that it is a problem of the robot itself and not depending on the software of this repository. The fact that the software might not work is because the operating system is not detecting the robot and not because the software needs a reboot in order to work :) That is why I don't think this comment has to stay in the repository. Nevertheless, if you think we can decide to add a "Known problems" section in the README which explains that if the software is not running it is necessary to check whether the operating system is detecting the robot. If not, the user should try rebooting the operating system.

Reply:
Can you please add a section regarding 'Known problems' as discussed above? Thank you